src/realm/object-store/shared_realm.hpp

```diff
@@ -238,6 +239,11 @@ class Realm : public std::enable_shared_from_this<Realm> {
     {
         return m_schema;
     }
+    Class get_class(StringData object_type)
+    {
+        return {shared_from_this(), object_type};
```

Comment:
It is probably better to get rid of this unsafe constructor from `Class` and instead have this method call `schema().find(object_type)`, validate the result (possibly by returning an `optional<Class>`), then call the safe `Class` constructor with the result.

Reply:
Probably simpler that way :-)